distribution/docs/src/main/resources/_contents/securing-contents.adoc

```diff
@@ -35,25 +46,35 @@ As you add REST endpoints, you may need to add different types of authentication
 |Description
 
 |`saml`
-|Activates single-sign on (SSO) across all REST endpoints that use.
+|Activates single-sign on (SSO) across all REST endpoints that use SAML.
 
 |`basic`
 |Activates basic authentication.
 
 |`PKI`
-|Activates public key infrastructure authentication
+|Activates public key infrastructure authentication.
 
 |`IdP`
-|
+|Activates SSO authentication through external Identity Providers
```

Comment:
Activates SAML Web SSO authentication support. Additional configuration is necessary.